fontir/src/ir.rs

```diff
@@ -444,6 +453,13 @@ impl Axis {
     }
 }
 
+/// IR for a named position in variation space
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Eq)]
+pub struct NamedInstance {
+    pub name: String,
+    pub location: NormalizedLocation,
```

Comment:
I think it's better if the ir for NamedInstance location has a UserLocation, not a NormalizedLocation. That's what fvar contains after all. It would be confusing if the ir had internal normalized location because then, without knowing exactly the context, one could interpret as either as default-normalized or after-avar-normalized.. Better to store as user-space locations.

Reply:
Oh good point, let me see about adjusting that.

Reply:
That makes fontbe fvar nicer too.

Reply:
Done.